app/views/devise/registrations/edit.html.erb

```diff
@@ -1,75 +1,85 @@
-<h2>Edit <%= resource_name.to_s.humanize %></h2>
-
-<%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
-  <%= devise_error_messages! %>
-
-  <div class="field">
-    <%= f.label :business_name %><br />
-    <%= f.text_field :business_name %>
-  </div>
-
-  <div class="field">
-    <%= f.label :address %><br />
-    <%= f.text_field :address %>
-  </div>
-
-  <div class="field">
-    <%= f.label :phone_number %><br />
-    <%= f.text_field :phone_number %>
-  </div>
-
-  <div class="field">
-    <%= f.label :contact_name %><br />
-    <%= f.text_field :contact_name %>
-  </div>
-
-  <div class="field">
-    <%= f.label :type_of_donor %><br />
-    <%= f.text_field :type_of_donor %>
-  </div>
-
-
-  <div class="field">
-    <%= f.label :email %><br />
-    <%= f.email_field :email, autofocus: true %>
+<div class="container">
+  <div id="logo-container"><img src="http://www.rescuingleftovercuisine.org/assets/RLC_LOGO-900fcd5d7d719b63e43d29ac23775f74f17100194e2690e17833a091b016da85.png" alt="Rescuing Leftover Cuisine Logo" style="width=302px;height:228px"></div>
```

Comment:
Hmmm, without the id on the logo container, the logo is centered and slightly over the text and with it it's off to the side (looks better), I'm going to aim to have the background on all the pages, so I may remove the logo entirely if that's okay?